### PR TITLE
Added claffy and Zhang to UCSD

### PR DESCRIPTION
**Pull request by snoeren**
# Contributing to CSrankings

Thanks for contributing to CSrankings! Here are some guidelines to getting your pull request accepted.

_Do not use Excel to edit any .csv files; Excel incorrectly tries to
convert some Google Scholar entries to formulas, corrupting the
database. Use a text editor like emacs or NotePad instead._

**Inclusion criteria**

- [x] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track
faculty members on a given campus who can *solely* advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department.

**Updating an affiliation or home page**

- [x] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings.csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put NOSCHOLARPAGE.

**Adding one or more faculty members (including an entire department)**

- [ ] If the department is not yet listed in CSrankings, the entire faculty needs to be added (not just one faculty member).

- [x] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings.csv`; include disambiguation suffixes like 0001 as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [ ] If DBLP has multiple entries for this person, all of them need to be listed. If an alias is not already present in `dblp-aliases.csv`, add it.

- [ ] If the institution you are adding is not in the US,
update `country-info.csv`.

### CODE REVIEW

File: csrankings.csv [hunk 1]
```diff
@@ -2004,6 +2007,7 @@ Behnaz Ghoraani,Florida Atlantic University,http://www.eng.fau.edu/directory/fac
 Behrooz A. Shirazi,Washington State University,https://school.eecs.wsu.edu/people/faculty/behrooz-a-shirazi,NOSCHOLARPAGE
 Behrooz Shirazi,Washington State University,https://school.eecs.wsu.edu/people/faculty/behrooz-a-shirazi,NOSCHOLARPAGE
 Behrouz Minaei-Bidgoli,Iran University of Science and Technology,http://ce.iust.ac.ir/content/544/Dr.-Behrouz-Minaei,M8tgU-wAAAAJ
+Behrouz Touri,University of California - San Diego,http://eceweb.ucsd.edu/~btouri/,W1BHaOwAAAAJ
```
Comment (emeryberger) [inline csrankings.csv]: Not on CS page: https://cse.ucsd.edu/people/faculty-profiles

File: csrankings.csv [hunk 2]
```diff
@@ -2095,6 +2099,7 @@ Bent Thomsen,Aalborg University,http://people.cs.aau.dk/~bt,mhyMmZoAAAAJ
 Benyuan Liu,University of Massachusetts Lowell,http://www.cs.uml.edu/~bliu,s2n1-HMAAAAJ
 Berk Burak Ustundag,Istanbul Technical University,http://akademi.itu.edu.tr/bustundag,JJY3eIEAAAAJ
 Berk Canberk,Istanbul Technical University,http://bcrg.itu.edu.tr,q3ueClIAAAAJ
+Bekr Ustun,University of California - San Diego,https://datascience.ucsd.edu/staff-members/berk-ustun/,6z_XWYcAAAAJ
```
Comment (emeryberger) [inline csrankings.csv]: Not on CS page.

File: csrankings.csv [hunk 3]
```diff
@@ -4678,11 +4686,13 @@ Don Cleland,Massey University,http://www.massey.ac.nz/massey/expertise/profile.c
 Don Fussell,University of Texas at Austin,http://www.cs.utexas.edu/users/fussell,NOSCHOLARPAGE
 Don Jardine,Queen’s University,http://www.cs.queensu.ca/people/faculty.php,NOSCHOLARPAGE
 Don Mitchell Wilkes,Vanderbilt University,http://engineering.vanderbilt.edu/bio/mitchell-wilkes,NOSCHOLARPAGE
+Don Norman,University of California - San Diego,http://www.jnd.org,h-2JebwAAAAJ
 Don Perlis,University of Maryland - College Park,https://www.cs.umd.edu/users/perlis,v2X9gX8AAAAJ
 Don S. Batory,University of Texas at Austin,https://www.cs.utexas.edu/~dsb,jUzA3H8AAAAJ
 Don Sheehy,North Carolina State University,http://donsheehy.net,A5D1DF4AAAAJ
 Don Towsley,University of Massachusetts Amherst,https://www.cics.umass.edu/faculty/directory/towsley_donald,KIFPVWoAAAAJ
 Donald A. Adjeroh,West Virginia University,https://www.statler.wvu.edu/faculty-staff/faculty/donald-a-adjeroh,NOSCHOLARPAGE
+Donald A. Norman,University of California - San Diego,http://www.jnd.org,h-2JebwAAAAJ
```
Comment (emeryberger) [inline csrankings.csv]: Not on CS page.

**Review by emeryberger — CHANGES_REQUESTED** [review]
It's not clear that these faculty can solely advise PhD students in CS, as they are not listed as CS faculty or affiliates.
https://cse.ucsd.edu/people/faculty-profiles

**Comment by snoeren** [comment]
Faculty in any of our departments can advise CSE students; these faculty are all in related areas.

- Alex

**Comment by snoeren** [comment]
To be explicit, must of these faculty are affiliated with our new Data Science institute, HDSI, which, to be blunt, will be the new hiring engine for CS at UCSD.

- Alex

**Comment by snoeren** [comment]
And since I see you call out three in particular, Touri is in ECE, Ustun is HDSI, and Norman is CogSci.  All are tenure track and so can advise CSE students.  HDSI in particular does not have PhD students of its own so must admit through other departments (generally CSE).

If you’re curious, the official mapping of title to ability to serve as a solo advisor is here.  The upshot is any tenure track (including teaching by blanket exception, which all CSE teaching professors have) professors can advise students in any department:

https://grad.ucsd.edu/academics/progress-to-degree/committees.html#Doctoral-Committee-Membership-T

- Alex

**Comment by emeryberger** [comment]
Thanks for the clarification. There some conflicts with `csrankings.csv`; please resolve and I will merge.

**Review by emeryberger — APPROVED** [review]

**Pull request merged**